lib/src/streams.js

```diff
@@ -0,0 +1,149 @@
+const addStreamsApi = matroid => {
+  // https://www.matroid.com/docs/api/index.html#api-Streams-PostStreams
+  matroid.createStream = function(url, name) {
```

Comment:
Let's update this to include the other parameters we have for this:
- detectionFps
- recordingEnabled
- retentionDays